### PR TITLE
PP-9796 OpenAPI specs for card types endpoint

### PR DESCRIPTION
**Pull request by kbottla**
## WHAT YOU DID
- OpenAPI specs for card types endpoint

### CODE REVIEW

File: openapi/connector_spec.yaml [hunk 1]
```diff
@@ -77,6 +88,34 @@ components:
...
+          example: ac8a3abd-bcfd-4fa6-8905-321ce913e7f5
+        label:
+          type: string
+          example: Visa
+        requires3ds:
+          type: boolean
+        type:
+          type: string
+          enum:
+          - CREDIT
```
Comment (SandorArpa) [inline openapi/connector_spec.yaml:109]: I think we can also have `CREDIT_OR_DEBIT` or null, if `CREDIT_OR_DEBIT`

**Review by SandorArpa — CHANGES_REQUESTED** [review]

File: openapi/connector_spec.yaml (hunk 1 above)
Comment (kbottla) [reply openapi/connector_spec.yaml:109]: `/v1/api/card-types` endpoint only returns list of card types (for admin tool listing). It is only CREDIT or DEBIT currently. On charges we can have `null` for the card type.

Comment (SandorArpa) [reply openapi/connector_spec.yaml:109]: 👍

**Review by SandorArpa — APPROVED** [review]
LGTM

**Pull request merged**